src/Vester.sol

```diff
@@ -247,6 +250,8 @@ contract Vester is Initializable, Pausable, IVester {
         if (!IMerkle(VOTING_REWARDS_MERKLE_STASH).isClaimed(token, index)) {
             /// Claim voting rewards.
             IMerkle(VOTING_REWARDS_MERKLE_STASH).claim(token, index, address(this), amount, proofs);
+        } else {
+            revert NoVotingRewards();
```

Comment:
Good point, I should have removed it! Not necessary here! 

The idea is that, through the Merkle, anyone can claim on behalf of the Vestor, thus the isClaimed check. If claimed, it would transfer the whole balance to the beneficiary.

Reply:
thanks, adjusted